src/com/android/kprofiles/battery/KprofilesSettingsFragment.java

```diff
@@ -115,21 +121,28 @@ public boolean onOptionsItemSelected(MenuItem item) {
     }
 
     private String modesDesc() {
-        String mode = getCurrentKProfilesMode();
         String descrpition = null;
-        switch (mode) {
-            case "0":
-                descrpition = getString(R.string.kprofiles_modes_none_description);
-                break;
-            case "1":
-                descrpition = getString(R.string.kprofiles_modes_battery_description);
-                break;
-            case "2":
-                descrpition = getString(R.string.kprofiles_modes_balanced_description);
-                break;
-            case "3":
-                descrpition = getString(R.string.kprofiles_modes_performance_description);
-                break;
+        if (iskProfilesModesSupported == true) {
```

Comment:
again == true, return instead of indent

Reply:
Thanks for your advices!
That's really an eye opening to me!